pkg/grpcutil/grpcutil.go

```diff
@@ -55,6 +60,20 @@ func (s SecurityConfig) ToTLSConfig() (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+// GetOneAllowedCN only gets the first one CN.
+func (s SecurityConfig) GetOneAllowedCN() (string, error) {
+	allowedCN := ""
+	i := len(s.CertAllowedCN)
+	switch {
+	case i > 1:
+		return "", errors.New("Currently only supports one CN")
```

Comment:
Do we need to return one and give the error?

Reply:
I think no need.